streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreChangelogReaderTest.java

```diff
@@ -386,9 +386,10 @@ public void shouldThrowTaskMigratedExceptionIfEndOffsetGetsExceededDuringRestore
     }
 
     private void setupConsumer(final long messages,
-                               final TopicPartition topicPartition) {
+                               final TopicPartition topicPartition,
```

Comment:
can we have an overload for `setupConsumer` without `startingOffset` that uses `0` as default ?